packages/ethereum-contracts/scripts/gov-create-new-app-registration-key.js

```diff
@@ -14,17 +16,34 @@ const {
  * @param {Address} options.from Address to deploy contracts from
  * @param {boolean} options.protocolReleaseVersion Specify the protocol release version to be used
  *
- * Usage: npx truffle exec scripts/create-new-app-registration-key.js : {DEPLOYER} {REGISTRATION_KEY}
+ * Note: the key itself doesn't have much meaning, it could be "stolen" from a broadcast tx anyway.
+ * But since it's bound to a deployer address, that doesn't really matter.
+ *
+ * Usage: npx truffle exec scripts/create-new-app-registration-key.js : {DEPLOYER} {REGISTRATION_KEY} [EXPIRATION_TS]
```

Comment:
REGISTRATION_KEY -> APP_KEY?

Reply:
I don't think so. The appKey is derived from it, as before:
```
    const appKey = web3.utils.sha3(
        web3.eth.abi.encodeParameters(
            ["string", "address", "string"],
            [
                "org.superfluid-finance.superfluid.appWhiteListing.registrationKey",
                deployer,
                registrationkey,
            ]
        )
    );
```

Reply:
yea, my bad.